dissect/target/loaders/acquire.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+
+import logging
+import tarfile as tf
+import zipfile as zf
+from pathlib import Path
+
+from dissect.target.filesystems.tar import TarFilesystem
+from dissect.target.loaders.dir import find_and_map_dirs
+from dissect.target.loaders.tar import TarSubLoader
+from dissect.target.loaders.zip import ZipSubLoader
+from dissect.target.target import Target
+
+log = logging.getLogger(__name__)
+
+FILESYSTEMS_ROOT = "fs"
+FILESYSTEMS_LEGACY_ROOT = "sysvol"
+
+
+class AcquireTarSubLoader(TarSubLoader):
+    """Tar implementation for parsing Acquire collects"""
+
+    @staticmethod
+    def detect(tarfile: tf.TarFile) -> bool:
+        for member in tarfile.getmembers():
+            if member.name.startswith((f"/{FILESYSTEMS_ROOT}/", f"{FILESYSTEMS_ROOT}/",
+                                           f"/{FILESYSTEMS_LEGACY_ROOT}/", f"{FILESYSTEMS_LEGACY_ROOT}/")):
+                return True
+        return False
+
+    def map(self, target: Target) -> None:
+        tar_fs = TarFilesystem(tarfile=self.tar).path()
+        map_acquire(target, tar_fs)
```

Comment:
This combined with the change in `dissect/target/loaders/dir.py` will still iterate the tarfile twice. Why not just move the logic from `dissect/target/loaders/tar.py` in here?

It may be a good idea to add a small benchmark test for loading a tar file in a separate PR before merging this one. You can take a look at https://github.com/fox-it/dissect.target/pull/1129 for inspiration.

Reply:
I just tested it a bit with the benchmark, and indeed the old implementation is quicker. I have moved the logic from the tar loader into the acquire loader to make sure it stays efficient. @Schamper do you agree with this approach?

Reply:
Yes, efficiency is more important in this case than saving a few lines of code.